### DISCUSSION TITLE
Different bandwidth between version 4.0 and 4.2

### DISCUSSION THREAD

**Discussion by theanhhds** [Motion-Project/motion#1402] [Q&A]
Hi,

So I tried 2 different versions of Motion, 4.0 and 4.2. It came to my attention that when streaming video over the internet, version 4.2 uses **much more** bandwidth than older version 4.0.

Edit: I used the same setting in the motion.conf file, from dimension to framerate etc.

Does anyone have explanation for this behavior? Thank you :D

**Comment by Mr-Dave** [comment]
There was a major revision to how the web interface is processed between those versions but fundamentally it is the same method.  Motion jpeg images are being sent.  

A couple of things you could try.  First, try version 4.3 since that is the latest release and validate it still occurs there. Next,  make sure that the image sizes and frame rates of the images provided is the same.
